### PR TITLE
Added module on styles

### PR DESCRIPTION
**Pull request by renvrant**

### CODE REVIEW

File: styles/01-css-approaches.mdx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+* Functional CSS approaches let you avoid writing redundant CSS
+
+---
+
+## CSS-in-JS
+
+* Approaches that move some part of the CSS process into JavaScript
+* Normally try to solve the problem of CSS being in the global scope
+* Can require more set up, but end up more flexible in the long term
+* Read into CSS Modules and Styled Components if this method is of interest
```
Comment (e-schultz) [inline styles/01-css-approaches.mdx]: Do you have a list of resources you've found useful?

Can include them here - or as part of the readme of this repo - a collection of additional reading / resources / etc.

File: styles/01-css-approaches.mdx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    font-size: 3rem;
+}
+```
+
+```notes
+https://tachyons.io/#style
+```
+---
+
+```javascript
```
Comment (e-schultz) [inline styles/01-css-approaches.mdx]: I like this general idea - but wondering if maybe taking an example that may need a few more classes, and could show a more 'traditional approach'

```js
const Menu = ({ items }) => (
    <ul className="menu-list">
        {items.map((item, i) => (
            <li className="menu-item" key={i}>{item.name}</li>
        ))
    </ul>
```

then could turn into

```js
const Menu = ({ items }) => (
    <ul className="flex flex-row f3">
        {items.map((item, i) => (
            <li className="pl4" key={i}>{item.name}</li>
        ))
    </ul>
```
and - then 'what would be your `menu-list / menu item css classes, become your react components`

```js
const Menu = ({ items }) => (
    <MenuList>
        {items.map((item, i) => (
            <MenuItem key={i}>{item.name}</MenuItem>
        ))
    </MenuList>
```
maybe that's going too fine grained? I just remember the thing that made tachyons click as an 'ah-hah' for me was 'ah, that class-name soup that I was wanting class menu-item for, becomes a component"

Comment (renvrant) [reply styles/01-css-approaches.mdx]: I will add this!

File: styles/01-css-approaches.mdx (hunk 1 above)
Comment (renvrant) [reply styles/01-css-approaches.mdx]: I'll add to the readme.

**Pull request merged**